docs/faqs.md

```diff
@@ -209,3 +210,32 @@ This is for historical reasons, when older versions were permitted the overwhelm
 There are different motivations
 - time and resources: adding the build time required by the test suite plus execution time can increase our building times significantly across the 100+ configurations.
 - ConanCenter is a service that builds binaries for the community for existing library versions, this is not an integration system to test the libraries.
+
+## What is the policy for supported python versions?
+
+`Python 2.7` and earlier is not supported by the ConanCenter, as it's already [EOL](https://www.python.org/doc/sunset-python-2/).
+
+`Python 3.5` and earlier is also not supported by the ConanCenter, as it's already [EOL](https://www.python.org/dev/peps/pep-0478/).
+
+Versions `Python 3.6+` onwards are currently supported by the infrastructure and the recipes.
+
+Our [docker images](https://github.com/conan-io/conan-docker-tools) use `Python 3.7.5+` ATM.
+
+Windows agents currently use `Python 3.6.7+`. macOS agents use version `Python 3.7.3+`.
+
+The version run by our agents and docker images is a subject to change, as security updates to the Python are released, or they enter EOL.
```

Comment:
I would move it to https://github.com/conan-io/conan-center-index/blob/master/docs/supported_platforms_and_configurations.md

If we need to update Python version, I would prefer something more centralized.

Reply:
sorry, but I have to disagree :)
the page you have linked documents configurations we're building packages for (compilers, operation systems, architectures, etc.), or profiles, in other words.
I don't believe Python configuration goes here. maybe, eventually, we will have a dedicated page on what tools are installed in CI (include python, perl, Windows SDK, etc.), but right now I think dedicated FAQ section is okay.